scripts/us_census/pep/Population_Estimate_by_ASR/README.md

```diff
@@ -0,0 +1,55 @@
+# US Census PEP: Population Estimates by Age, Sex and Race
+
+## About the Dataset
+This dataset has Population Estimates for the National, State and County geographic levels in United States from the year 1900 to 2020 on a yearly basis.
+
+The population is categorized by various set of combinations as below:
+        
+        1. Age, Sex.
+        2. Age, Race.
+        3. Age, Sex, Race.
+
+### Download URL
+The data in txt/csv formats are downloadable from within https://www2.census.gov/programs-surveys/popest/tables and https://www2.census.gov/programs-surveys/popest/datasets. The actual URLs are listed in national/state/county JSON files.
+
+
+#### API Output
+These are the attributes that we will use
+| Attribute      					| Description                                                 				|
+|-------------------------------------------------------|---------------------------------------------------------------------------------------|
+| Year       					| The Year of the population estimates provided. 				|
+| Age   				| The Individual Ages or Age Buckets of the population in the US. 						|
+| Race   	| Races of the population in the US.  	|
+| Sex   				| Gender either Male or Female. 							|
+
+
+
+#### Cleaned Data
+Cleaned data will be inside [Output/USA_Population_ASRH.csv] as a CSV file with the following columns.
+
+- Year
```

Comment:
These should all either start with a capital letter or all be small

Reply:
changed the file names to be all small letters, it was made capitals earlier as they were short forms.